docs/source/add_new_data_type.md

```diff
@@ -0,0 +1,91 @@
+| Quantization (`scheme`) | `sym`, `asym`| The quantization scheme used for the new data type. `sym` represents symmetric quantization, `asym` represents asymmetric quantization.|
+| Quantization Granularity (`granularity`)| `per_channel`, `per_tensor`| The granularity at which quantization is applied. `per_channel` represents that the quantization is applied independently per channel, `per_tensor` represents that the quantization is applied to the entire tensor as a whole. |
+| Calibration Algorithm (`algorithm`)| `minmax`, `kl`| 	The calibration algorithm used for the new data type. `minmax` represents the minimum-maximum algorithm, `kl` represents the Kullback-Leibler divergence algorithm. |
+
+
+To add  4-bit quantization for `Conv2d` in the PyTorch backend. We can modify the `neural_compressor/adaptor/pytorch_cpu.yaml` as follows:
+
+```diff
+  ...
+  fp32: ['*'] # `*` means all op types.
+  
+
++    int4: {
++        'static': {
++            'Conv2d': {
++                'weight': {
++                    'dtype': ['int4'],
++                    'scheme': ['sym'],
++                    'granularity': ['per_channel'],
++                    'algorithm': ['minmax']},
++                'activation': {
++                    'dtype': ['uint4'],
++                    'scheme': ['sym'],
++                    'granularity': ['per_tensor'],
++                    'algorithm': ['minmax']},
++            },
++        }
++    }
+
+  int8: &1_11_capabilities {
+    'static': &cap_s8_1_11 {
+          'Conv1d': &cap_s8_1_11_Conv1d {
+  ...
+
+```
+The code states that the PyTorch Conv2d Operator has the ability to quantize weights to int4 using the `torch.per_channel_symmetric` quantization scheme, , with the supported calibration algorithm being `minmax`. Additionally, the operator can quantize activations to `uint4` using the `torch.per_tensor_symmetric` quantization scheme, with the supported calibration algorithm also being `minmax`.
+
+> Note: more details about the framework YAML can be found [here](./framework_yaml.md).
+
+
+### Use the New Data Type
+
+Once the new data type has been added to INC, it can be used in the same way as any other data type within the framework. To specify that all `Conv2d` operators should utilize 4-bit quantization, here's an example of how to do it::
+
+```python
+from neural_compressor.config import PostTrainingQuantConfig
+op_type_dict = {
+    'Conv2d': {
+        'weight': {
+            'dtype': ['int4']
+        },
+        'activation': {
+            'dtype': ['uint4']
+        }
+    }
+}
+conf = PostTrainingQuantConfig(op_type_dict=op_type_dict)
+...
+
+```
+
+With this code, all `Conv2d` operators will be quantized to 4-bit, with weight using `int4` and activation using `uint4`.
+
+
+### Others
+It's important to note that INC currently only supports N-bit quantization for the PyTorch backend, where N is an integer between 1 and 7. While other backends or quantization schemes may be added in the future.
```

Comment:
what we should mention is the flexible design of INC to easily extend to other new data type.

Reply:
and pls add a link to `advance` page for user reference. 

The title of this doc should be something like "How to support new data type, like int4, with few line changes". 

the whole idea should highlight our extensible design like I commented at above.